packages/braintree-integration/src/braintree-paypal/braintree-paypal-credit-customer-strategy.ts

```diff
@@ -98,6 +98,7 @@ export default class BraintreePaypalCreditCustomerStrategy implements CustomerSt
                 braintreepaypalcredit,
                 methodId,
                 Boolean(config.testMode),
+                { ...checkoutTopButtonStyles, height: 36 },
```

Comment:
Are we hardcoding 36 here? What about `DefaultCheckoutButtonHeight`?

Reply:
Updated